packages/components/select-v2/src/defaults.ts

```diff
@@ -27,7 +28,7 @@ import type {
   PopperEffect,
 } from '@element-plus/components/popper'
 
-export const SelectProps = buildProps({
+export const selectV2Props = buildProps({
```

Comment:
These variables are exported and may be used by users, so we should not modify the variable names directly.